apis/Google.Cloud.Logging.Console/Google.Cloud.Logging.Console/GoogleCloudConsoleFormatter.cs

```diff
@@ -177,6 +185,35 @@ private static void WriteKeyValuePairs(Utf8JsonWriter writer, IEnumerable<KeyVal
         writer.WriteEndObject();
     }
 
+    private void MaybeWriteTraceInformation(Utf8JsonWriter writer)
+    {
+        if (string.IsNullOrWhiteSpace(_options.TraceGoogleCloudProjectId))
+        {
+            return;
+        }
+
+        var activity = Activity.Current;
+        if (activity == null)
```

Comment:
Massively minor nit, doesn't even need fixing here - we generally prefer `is null` over `== null` these days.

Reply:
And with a further tweak you can combine the two ifs:

```suggestion
        if (string.IsNullOrEmpty(_options.TraceGoogleCloudProjectId) || Activity.Current is not Activity activity)
        {
            return;
        }
```

Reply:
Done.